Configuration/Generator/python/TTbar_Pow_LHE_13TeV_cff.py

```diff
@@ -7,6 +7,7 @@
     outputFile = cms.string('cmsgrid_final.lhe'),
     scriptName = cms.FileInPath('GeneratorInterface/LHEInterface/data/run_generic_tarball_cvmfs.sh'),
     generateConcurrently = cms.untracked.bool(True),
+    postGenerationCommand = cms.untracked.vstring('mergeLHE.py', '-i', 'thread*/cmsgrid_final.lhe', '-o', 'cmsgrid_final.lhe'),
```

Comment:
What will happen if we run crab jobs with generateConcurrently set to True?

Reply:
Thanks @qliphy . Indeed before the "mergeLHE.py" script is merged into cmssw, the crab job may fail - it is pretty weird to me but the fact is that crab cannot find the path of "mergeLHE.py" if it is in the local cmssw `src/`. But I checked that after the script is merged, the issue will be resolved.